src/tools/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
-add_library(regression_main regression_fixture.cpp)
-target_link_libraries(regression_main gtest gridtools)
+if(ENABLE_TESTING)
+  add_library(regression_main regression_fixture.cpp)
+  target_link_libraries(regression_main gtest gridtools)
```

Comment:
I think we have a mixed convention, but if I add new CMake code, I use four spaces. Can we agree on a convention?

Reply:
4 is find with me. I used the convention from directory up.